exporters/sql_export/README.rst

```diff
@@ -0,0 +1,281 @@
+*************************************
+Eksport til Actual State SQL database
+*************************************
+
+Indledning
+==========
+Denne eksport laver et dags-dato udtræk af MO og afleverer det i en SQL database.
+
+For at opnå den nødvendige afviklingshastighed, tilgår eksporten data direkte fra
+LoRa hvor det er muligt at lave bulk udtræk, baseret på den kendte datamodel for
+OS2MO behandles de udtrukne data så SQL eksporten for et udseende som svarer til
+det man finder i MO.
+
+Tabellerne er for den praktiske anvendeligheds skyld ikke 100% normaliserede, der
+vil således for alle klasser altid være både en reference til primærnøglen for
+klassen plus en tekstrepræsentation, så det er muligt at aflæse tabellen uden at
+skulle foretage et join mod tabellen ``klasser`` for alle opslag.
+
+Implementerigen er foretaget ved hjælp af værktøjet SQLAlchemy, som sikrer at
+det er muligt at aflevere data til en lang række forskellige databasesystemer,
+det er desuden muligt at køre hele eksporten mod en flad SQLite fil som muliggør
+eksporting helt uden en kørende databaseserver. 
+
+Konfiguration
+=============
+
+For at anvende eksporten er det nødvendigt at oprette et antal nøgler i
+`settings.json`:
+
+ * ``exporters.actual_state.manager_responsibility_class``: UUID på det lederansvar,
```

Comment:
Har vi ikke en nøgle svarende til den for den rapport, der endnu ikke er i cron hos viborg - den der med lederspænd?
undskyld at jeg ikke har fået den i cron.